docker/docker-compose.yml

```diff
@@ -38,7 +37,8 @@ services:
       - VIRTUAL_HOST=ddb-proxy.example.com
       - VIRTUAL_PORT=3000
       - LETSENCRYPT_HOST=ddb-proxy.example.com
+
 networks:
   default:
-    external:
-      name: nginx-proxy
+    name: nginx-proxy
+    # external: true
```

Comment:
Definition for external networks has been changed. External now should be used as a flag not as a parent. I also disabled it since it is in my opinion not necessary here, since all modules are in the same `docker-compose` file